app/Livewire/Project/Application/PreviewsCompose.php

```
@@ -28,10 +35,9 @@ public function save()
         try {
             $this->authorize('update', $this->preview->application);
 
-            $domain = data_get($this->service, 'domain');
             $docker_compose_domains = data_get($this->preview, 'docker_compose_domains');
```

PHPMD (2.15.0)

The variable $docker_compose_domains is not named in camelCase. (undefined)

(CamelCaseVariableName)